### PR TITLE
Initial DayZ Documentation

### PR DESCRIPTION
**Pull request by Eutyches**
You might also wish to change the name of the repository to "arma3-dayz-unix-launcher."

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
+      * [DayZ](#build-process-for-dayz)
 * [Launch parameters](#launch-parameters)
 * [Sceenshots](#screenshots)
+* [DayZ Installation Tips](#dayz-installation)
 
 ## Installing
 ### From package
 
-For Debian based distributions (Debian, Ubuntu), Arch based distributions (Arch, Manjaro) and Mac OS X there are packages available in [releases tab!](https://github.com/muttleyxd/arma3-unix-launcher/releases)
+For Debian based distributions (Debian, Ubuntu), Arch based distributions (Arch, Manjaro), and MacOS there are packages available in [releases tab!](https://github.com/muttleyxd/arma3-unix-launcher/releases). Choose either Arma 3 or DayZ.
```
Comment (muttleyxd) [inline README.md:23]: If we're finally updating this I would mention AppImage as it works anywhere where we have GNU/Linux.

```
There are GNU/Linux and macOS packages available in [releases tab!](https://github.com/muttleyxd/arma3-unix-launcher/releases).
For GNU/Linux following package types are available:
- pkg.tar.xz package (for Arch Linux, Manjaro and few others) (arma3-unix-launcher)
- deb package for Ubuntu 16.04 and 18.04 (arma3-unix-launcher)
- dmg package for macOS (arma3-unix-launcher)
- AppImage packages for any GNU/Linux running glibc 2.23 or newer (arma3-unix-launcher and dayz-linux-launcher)
```

**Review by muttleyxd — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
 
-For Arch based distributions there's an AUR package available - it's called `arma3-unix-launcher-bin`
+For Arch based distributions there's an AUR package available (for Arma 3). It iss called `arma3-unix-launcher-bin`.
```
Comment (graves501) [inline README.md:25]: `It iss called` -> `It is called`

**Comment by muttleyxd** [comment]
@Eutyches I plan to merge it soon. Will you have some time to take a look at changes we have proposed?

**Comment by Eutyches** [comment]
> @Eutyches I plan to merge it soon. Will you have some time to take a look at changes we have proposed?

Updated with your suggested changes and a final editorial pass. Pardon the delay. Thank you again :smile:

**Pull request merged**